src/patterns/cookies-page/index.md

```diff
@@ -99,6 +124,14 @@ When the user sets or changes their cookie preferences, use a green [notificatio
 
 {{ example({ group: "patterns", item: "cookies-page", example: "cookies-updated", html: true, nunjucks: true, open: false }) }}
 
+<div class="app-wcag-22" id="wcag-avoid-cookie-preferences-reentry" role="note">
+  {{ govukTag({
+    text: "WCAG 2.2",
+    classes: "app-tag"
+  }) }}
+  <p>Save cookie preferences for returning users whenever technically possible, so they do not need to enter their preferences again. This relates to WCAG 2.2 success criterion <a href="https://www.w3.org/WAI/WCAG22/Understanding/redundant-entry.html">3.3.7 Redundant Entry</a>.</p>
+</div>
```

Comment:
Remove Redundant entry message for Cookies page. 
Not WCAG failure, also overall message to save settings is covered in Cookie banner.

```suggestion
```